docs/apm.md

```diff
@@ -9,37 +9,45 @@ hide_title: true
 
 **Upgradeability** is one of the key features of [aragonOS](/docs/aragonos-intro.html).
 
-We decided to build APM as the main way to **distribute different versions of the packages** that comprise Aragon Core. A package or **repository** (repo) in the APM keeps track of evolving versions of its contents (the webapp component typically) and smart contract code (if applicable).
+Because upgradeability implies there will be multiple versions of a package of software, we decided to build the Aragon Package Manager (APM) as the main way to **distribute different versions of the packages** that comprise the Aragon client. As we built it, however, we realized that its use cases could extend far beyond just ours as a publicly accessible piece of infrastructure living on Ethereum.
 
-## Architecture: Registries, repos and versions
+A package or **repository** (repo) in an APM registry keeps track of evolving versions of its contents (the webapp component typically) and smart contract code (if applicable).
 
-Each APM Registry is a smart contract that owns an [Ethereum Name Service](https://ens.domains/) (ENS) domain and therefore can create subdomains for its repos. This gives users the ability to **find repos with human-readable names**.
+## Architecture: registries, repos and versions
 
-We envision that many APM registries will be created for different purposes and types of packages. For Aragon, we use the `aragonpm.eth` registry to host our core components. We also have the `open.aragonpm.eth` registry for community developed packages, where anyone can create repos.
+![](/docs/assets/apm-arch.png)
+> The architecture of an APM DAO
```

Comment:
Hmm, yeah, on some level it might. cc @cjyabraham should we not use this styling for image notes?

Reply:
We currently don't have a way to style image captions and notes differently than warning boxes.  For notes and warnings, we use a title to introduce them so it's clear.  Captions usually don't have a title.  I think this might be ok for now but I have [made a note](https://docs.google.com/document/d/1B1nyb95LgBtrWY8Av0FnvnpUfvOb13wfjGZb28svSVs/edit#) that we need to stylistically treat them differently.  We can look at this next month when I work with Delfi on the redesign.